services/api-server/src/simcore_service_api_server/utils/client_base.py

```diff
@@ -100,20 +51,23 @@ def setup_client_instance(
     assert issubclass(api_cls, BaseServiceClientApi)  # nosec
 
     # Http client class
-    client_class: type = httpx.AsyncClient
+    client: httpx.AsyncClient = httpx.AsyncClient(base_url=api_baseurl)
     with suppress(AttributeError):
         # NOTE that this is a general function with no guarantees as when is going to be used.
         # Here, 'AttributeError' might be raied when app.state.settings is still not initialized
         if capture_path := app.state.settings.API_SERVER_DEV_HTTP_CALLS_LOGS_PATH:
-            _setup_capture_logger_once(capture_path)
-            client_class = _AsyncClientForDevelopmentOnly
+            from .client_base_dev import AsyncClientForDevelopmentOnly
```

Comment:
are you sure relative imports inside functions is actually allowed? My guess is that this code is not test-covered :-)

Reply:
I changed it to an absolute import and checked that it works.